pkg/services/featuremgmt/registry.go

```diff
@@ -126,5 +126,25 @@ var (
 			Description: "Highlight Enterprise features",
 			State:       FeatureStateStable,
 		},
+		{
+			Name:        "liveDiscussions",
```

Comment:
Is this one necessary?  liveDashboardDiscussions | liveAnnotationDiscussions?

Reply:
Done